src/pages/TopUp/index.tsx

```diff
@@ -184,29 +193,40 @@ const _TopUpPage: React.FC<Props> = () => {
       {/* the wallet connect modal controls it's own display, so it is always rendered here */}
       <WalletConnectModal />
 
-      <PageTemplate title="Top up a validator">
+      <PageTemplate
+        title={formatMessage({ defaultMessage: 'Top up a validator' })}
+      >
         {/* render a "back button" if there's a selected validator */}
         {selectedValidator && (
           <BackText
             onClick={() => setSelectedValidator(null)}
             color="blueMedium"
           >
             <Arrow />
-            All Validators
+            <FormattedMessage defaultMessage="All validators" />
           </BackText>
         )}
 
         <SubTextContainer className="mt20">
           <Text className="mt10">
-            If your validator’s balance is getting low because you haven’t been
-            active for a while, you may wish to top-up you balance to ensure
-            your validator isn’t ejected from the validator set for having a
-            balance that is too low. Low-balance ejections occur when a
-            validator’s effective balance drops below 16 ETH. The Launchpad only
-            facilitates top-ups that get validators’ balances to 32 ETH, as
-            validator rewards are based on effective balance which is capped at
-            32 ETH.
+            <FormattedMessage
+              defaultMessage="You may need to top up your validator's balance for two important reasons. If your validator's effective balance is below {PRICE_PER_VALIDATOR} ETH you won't be earning your full staker rewards. And if it drops as low as {EJECTION_PRICE} ETH the system will eject your validator."
+              values={{ PRICE_PER_VALIDATOR, EJECTION_PRICE }}
+            />
           </Text>
+          <Alert variant="info" className="my20">
+            <FormattedMessage
+              defaultMessage="{PRICE_PER_VALIDATOR} ETH is the maximum effective validator balance. This means you won't earn more rewards if your validator's effective balance goes above {PRICE_PER_VALIDATOR}. However you will earn less if it dips below {PRICE_PER_VALIDATOR}."
```

Comment:
```suggestion
              defaultMessage="{PRICE_PER_VALIDATOR} ETH is the maximum effective validator balance. This means you won't earn more rewards if your validator's balance goes above {PRICE_PER_VALIDATOR}. However you will earn less if it dips below {PRICE_PER_VALIDATOR}."
```

Reply:
@ryancreatescopy slightly tweaked this. 

> This means you won't earn more rewards if your validator's effective balance goes above 32 ETH

It's not possible for the *effective* balance to go above 32 ETH.

Reply:
:+1: